src/client/Microsoft.Identity.Client.Extensions.Msal/Microsoft.Identity.Client.Extensions.Msal.csproj

```
@@ -11,8 +11,16 @@
     <!--This will generate AssemblyVersion, AssemblyFileVersion and AssemblyInformationVersion-->
     <Version>$(MsalClientSemVer)</Version>
     <Description>This package contains the public client (desktop) caching to Microsoft Authentication Library for .NET (MSAL.NET)</Description>
+    <PackageReadmeFile>README.md</PackageReadmeFile>
   </PropertyGroup>
 
+  <ItemGroup>
+    <None Include="..\..\..\README.md">
```

```suggestion
    <None Include="..\..\..\README.md">
      <Pack>True</Pack>
      <PackagePath>\</PackagePath>
    </None>
```
<None Include="..\..\..\README.md" Pack="true" PackagePath="\" />